source/index.txt

```diff
@@ -1,80 +1,260 @@
+            :ref:`mongosh-read`.
+
+         .. step:: Update Documents
+
+            :binary:`~bin.mongosh` supports common update operations, 
+            including:
+
+            - :method:`db.collection.updateOne()`
+            - :method:`db.collection.updateMany()`
+            - :method:`db.collection.replaceOne()`
+
+            For more information and examples, refer to :ref:`mongosh-update`.
+
+         .. step:: Delete Documents
+
+            :binary:`~bin.mongosh` supports common delete operations, 
+            including:
+
+            - :method:`db.collection.deleteMany()`
+            - :method:`db.collection.deleteOne()`
+
+            For more information and examples, refer to :ref:`mongosh-delete`.
+
+         .. step:: Run Aggregation Pipelines
+
+            You can run aggregation pipelines in :binary:`~bin.mongosh` 
+            using the :method:`db.collection.aggregate()` method. 
+            Aggregation pipelines transform your documents into 
+            aggregated results based on the stages you specify. For more 
+            information and examples, refer to :ref:`mdb-shell-aggregation`.
+
+      .. image:: /images/mongosh-plp/Technical_ACTION_SearchDocument_Thumbnail_BS_Mist.png
+         :alt: Abstract illustration showing CRUD and aggregation operations
+
+   .. tab:: Administer and Manage
+      :tabid: administer
+
+      .. procedure::
+
+         .. step:: Manage Databases and Collections
+
+            View information about databases, create collections or views,
+            or drop databases - all from your shell. See all 
+            :ref:`Database Methods <mongosh-database-methods>`.
+
+            Perform collection operations, create or delete indexes, or 
+            explain queries with :ref:`Collection Methods 
+            <mongosh-collection-methods>`.
+
+         .. step:: Administer Servers
+
+            Manage :ref:`replication <mongosh-replication-methods>` or 
+            :ref:`sharding <mongosh-sharding-methods>` conveniently in 
+            your shell.
+
+            Check server status with a variety of :ref:`Server Status Methods 
+            <mongosh-server-status-methods>`.
+
+         .. step:: Manage Users and Roles
+
+            Create or update roles, define and update privileges, or drop 
+            roles using :ref:`Role Management Methods 
+            <mongosh-role-management-methods>`.
+
+            Create and update users, authenticate users, and manage user
+            roles with :ref:`User Management Methods 
+            <mongosh-user-management-methods>`. 
+
+      .. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(1)_Spot.png
+         :alt: Abstract illustration showing a terminal performing operations
+
+   .. tab:: Automate with Scripts
+      :tabid: scripts
+
+      .. procedure::
+
+         .. step:: Run Scripts for CRUD or Administrative Tasks
+
+            Write scripts for the |mdb-shell| that perform CRUD or 
```

Comment:
Reworded a bit for clarity:
"Write scripts to run with the MongoDB Shell that perform CRUD or administrative operations in MongoDB"